benchmarks/test_peer_selection.py

```diff
@@ -22,81 +22,78 @@
     absolute_import, unicode_literals, print_function, division
 )
 
-import mock
-import random
+from tornado import gen, ioloop
```

Comment:
Are the perf numbers in the summary master vs. this branch running old vs. new benchmark or are they both running this benchmark? Need apples-to-apples comparisons.